hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -581,6 +581,13 @@ public final class ScmConfigKeys {
 
   public static final String OZONE_AUDIT_LOG_DEBUG_CMD_LIST_SCMAUDIT =
       "ozone.audit.log.debug.cmd.list.scmaudit";
+
+  public static final String OZONE_SCM_CHECK_EMPTY_CONTAINER_ON_DISK_ON_DELETE =
```

Comment:
Do we need a description for this in `ozone-default.xml`?

Reply:
Added

Reply:
I don't think we need to add this config to ozone-default.xml we do not expect outside of debug scenarios to flip this config option.